mlir/include/mlir/Dialect/OpenMP/OpenMPClauseOperands.h

```diff
@@ -32,37 +32,37 @@ namespace omp {
 
 struct AlignedClauseOps {
   llvm::SmallVector<Value> alignedVars;
-  llvm::SmallVector<Attribute> alignmentAttrs;
+  llvm::SmallVector<Attribute> alignments;
 };
 
 struct AllocateClauseOps {
-  llvm::SmallVector<Value> allocatorVars, allocateVars;
+  llvm::SmallVector<Value> allocateVars, allocatorVars;
 };
 
 struct CancelDirectiveNameClauseOps {
-  ClauseCancellationConstructTypeAttr cancelDirectiveNameAttr;
+  ClauseCancellationConstructTypeAttr cancelDirective;
 };
 
 struct CollapseClauseOps {
-  llvm::SmallVector<Value> loopLBVar, loopUBVar, loopStepVar;
+  llvm::SmallVector<Value> collapseLowerBound, collapseUpperBound, collapseStep;
```

Comment:
Yes, I added the "collapse" prefix in reference to the clause name. Leaving them in singular was an oversight, thank you for pointing that out.

I get your point, but I'd prefer to try prefixing their name with something. I just want to avoid some "steps" clause to appear somewhere else and potentially collide. What do you think about `collapseAffectedLowerBounds`, `collapseAffectedUpperBounds`, `collapseAffectedSteps`?

Reply:
That indeed sounds better. I leave it up to you what you feel is less work.

For me either one means I will have to rebase my work again anyway.